packages/ra-core/src/controller/useSortState.ts

```diff
@@ -50,9 +55,7 @@ const sortReducer = (state: Sort, field: string | Sort): Sort => {
  * @param {string} initialSort.reference The linked resource name
  * @returns {SortProps} The sort props
```

Comment:
the jsDoc misses the way to call the `setSort` function

Reply:
done